src/ui/Misc/FileTypes.ts

```diff
@@ -70,9 +75,13 @@ export class FileTypes {
       // The main dictionary by using the original value.
       localizedString = l(filetype);
     }
+
+    const iconClass = StringUtils.htmlEncode(loweredCaseFiletype).replace(' ', '-');
```

Comment:
I think I would generally prefer using a `replace` before `htmlEncode`, although I don't see in this particular case how somebody could make use of spaces and dash in an XSS attack.